lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -2042,6 +2059,20 @@ void GrandCentralPass::runOnOperation() {
   // then the top-level instantiate interface will be marked for extraction via
   // a SystemVerilog bind.
   SmallVector<sv::InterfaceOp, 2> interfaceVec;
+  SmallDenseMap<FModuleLike, SmallVector<Operation *>> companionToInterfaceMap;
+  auto compareInterfaceSignal = [&](Operation *lhs, Operation *rhs) {
+    // If its a verbatim op, no need to check the string, because the interface
+    // names might not match. As long as the signal ops match that is
+    // sufficient.
+    if (isa<sv::VerbatimOp>(lhs) && isa<sv::VerbatimOp>(rhs))
+      return true;
```

Comment:
This check _could_ be erroneously true. To actually fix this would require a pretty big refactor of how `traverseBundle` works. Can you add a TODO and open a tracking issue once this lands?